mlops/london_taxi/src/mlops_pipeline.py

```diff
@@ -77,208 +58,83 @@ def london_taxi_data_regression(pipeline_job_input, model_name, build_reference)
+
+        # Set the environment name to custom environment using name and version number
+        prepare_data.environment = self.environment_name
+        transform_data.environment = self.environment_name
+        train_model.environment = self.environment_name
+        predict_result.environment = self.environment_name
+        score_data.environment = self.environment_name
+        register_model.environment = self.environment_name
+
+        gl_pipeline_components.append(prepare_data)
+        gl_pipeline_components.append(transform_data)
+        gl_pipeline_components.append(train_model)
+        gl_pipeline_components.append(predict_result)
+        gl_pipeline_components.append(score_data)
+        gl_pipeline_components.append(register_model)
+
+        pipeline_job = london_taxi_data_regression(
+            Input(type="uri_folder", path=registered_data_asset.id),
+            self.model_name,
+            self.build_reference,
         )
 
-        print(f"The job {pipeline_job.name} has been submitted!")
-        if output_file is not None:
-            with open(output_file, "w") as out_file:
-                out_file.write(pipeline_job.name)
-
-        if wait_for_completion == "True":
-            total_wait_time = 3600
-            current_wait_time = 0
-            job_status = [
-                "NotStarted",
-                "Queued",
-                "Starting",
-                "Preparing",
-                "Running",
-                "Finalizing",
-                "Provisioning",
-                "CancelRequested",
-                "Failed",
-                "Canceled",
-                "NotResponding",
-            ]
-
-            while pipeline_job.status in job_status:
-                if current_wait_time <= total_wait_time:
-                    time.sleep(20)
-                    pipeline_job = client.jobs.get(pipeline_job.name)
-
-                    print("Job Status:", pipeline_job.status)
-
-                    current_wait_time = current_wait_time + 15
-
-                    if (
-                        pipeline_job.status == "Failed"
-                        or pipeline_job.status == "NotResponding"
-                        or pipeline_job.status == "CancelRequested"
-                        or pipeline_job.status == "Canceled"
-                    ):
-                        break
-                else:
-                    break
-
-            if pipeline_job.status == "Completed" or pipeline_job.status == "Finished":
-                print("job completed")
-            else:
-                raise Exception("Sorry, exiting job with failure..")
-    except Exception as ex:
-        print(
-            "Oops! invalid credentials or error while creating ML environment.. Try again...",
-            ex,
-        )
-        raise
+        # demo how to change pipeline output settings
+        pipeline_job.outputs.pipeline_job_prepped_data.mode = "rw_mount"
```

Comment:
What does this do?

Reply:
Sets the storage mount type.. This one is read-write. Here's all the options: https://learn.microsoft.com/en-us/azure/machine-learning/how-to-read-write-data-v2?view=azureml-api-2&tabs=python#modes

Reply:
It's also possible to add this mount type in the component YAML files like this:
```
$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
name: predict_taxi_fares
version: 1
display_name: PredictTaxiFares
type: command
inputs:
  model_input:
    type: mlflow_model
  test_data:
    type: uri_folder
outputs:
  predictions:
    type: uri_folder
    mode: rw_mount
environment:  azureml:AzureML-sklearn-1.1-ubuntu20.04-py38-cpu@latest
code: ./../../../
command: >-
  python -m src.london_src.predict.predict
  --model_input ${{inputs.model_input}} 
  --test_data ${{inputs.test_data}}
  --predictions ${{outputs.predictions}}
```

Reply:
I guess my question is do we need to hardcode it in this spot or should it be handled in a config file?

Reply:
This is how it was so I say we leave it hardcoded for this PR. I think that's a good idea and not just for that property but also the others like default_datastore, default_compute, etc. Would you be up for creating a new GH issue on the DSToolkit repo?

Reply:
I think i'm slow and don't know how to make an issue

Reply:
Oh apparently I already made one so we're good to go!

https://github.com/microsoft/dstoolkit-mlops-v2/issues/147

Reply:
I'll make one :)

Reply:
https://github.com/microsoft/dstoolkit-mlops-v2/issues/185